src/js/Controllers/DisplayCapabilities.js

```diff
@@ -544,6 +544,65 @@ let capabilities = {
 			"driverDistractionCapability": {
 				"menuLength": 3,
 				"subMenuDepth": 2
+			},
+			videoStreamingCapability: {
+				preferredResolution: {
+					resolutionWidth: 960,
```

Comment:
if the scss master-height/width variables can be imported into react, it would be cool if those config settings would determine the supported resolutions.

Reply:
done in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/6125f5abb20538a97e15a641fabaa88dc9a1c3de